package.json

```
@@ -44,7 +44,9 @@
     "migration:fake": "yarn migration:run:fake && migration:revert:fake",
     "migration:run:fake": "npx typeorm-ts-node-commonjs  migration:run --fake",
     "migration:revert:fake": "npx typeorm-ts-node-commonjs migration:revert --fake",
-    "migration:check": "npx typeorm-ts-node-commonjs migration:generate src/migrations/migrations --check -d src/plugins/datasource.ts"
+    "migration:check": "npx typeorm-ts-node-commonjs migration:generate src/migrations/migrations --check -d src/plugins/datasource.ts",
+    "openapi:generate": "node dist/scripts/generateOpenAPI.js",
+    "openapi:lint": "vacuum lint ./openapi.json"
```

Shouldn't it be `.openapi.json`?  I have a personal preference for `openapi.json` though.

```suggestion
    "openapi:lint": "vacuum lint ./.openapi.json"
```